libticonv/trunk/tests/torture_ticonv.c

```diff
@@ -63,11 +62,7 @@ int main(int argc, char **argv)
     ticonv_iconv_t ticonv_iconv_instance;
 
     // ticonv.h
-    PRINTF(
-#ifdef __WIN32__
-    (unsigned long)
-#endif
-    ticonv_utf16_strlen, SIZE, NULL);
+    PRINTF(ticonv_utf16_strlen, SIZE, NULL);
```

Comment:
This change breaks the build on Debian's current MinGW cross-compiler:
```
$ i686-w64-mingw32-g++ --version
i686-w64-mingw32-g++ (GCC) 13-win32
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

...

torture_ticonv.c: In function ‘main’:
torture_ticonv.c:11:17: error: format ‘%ld’ expects argument of type ‘long int’, but argument 4 has type ‘size_t’ {aka ‘unsigned int’} [-Werror=format=]
   11 | fprintf(stderr, "%d\t" TYPE "\n", __LINE__, FUNCTION(__VA_ARGS__))
      |                 ^~~~~~
torture_ticonv.c:65:5: note: in expansion of macro ‘PRINTF’
   65 |     PRINTF(ticonv_utf16_strlen, SIZE, NULL);
      |     ^~~~~~
```

Reply:
fixed after testing things together via IRC.